ports/geckolib/gecko_bindings/sugar/ns_style_coord.rs

```diff
@@ -29,6 +30,25 @@ impl nsStyleUnion {
     }
 
     pub unsafe fn get_calc(&self) -> nsStyleCoord_CalcValue {
-        (*(*self.mPointer.as_ref() as *const nsStyleCoord_Calc))._base
+        (*self.as_calc())._base
+    }
+
+    pub unsafe fn addref_opt(&mut self, unit: &nsStyleUnit) {
+        if *unit == nsStyleUnit::eStyleUnit_Calc {
+            Gecko_AddRefCalc(self.as_calc_mut());
+        }
+    }
+
+    pub unsafe fn as_calc_mut(&mut self) -> &mut nsStyleCoord_Calc {
```

Comment:
Do `as_calc_mut` and `as_calc` need to be `pub`?  Looks like they're not called from elsewhere.